Source/WebKit/NetworkProcess/Classifier/WebResourceLoadStatisticsStore.cpp

```diff
@@ -449,23 +448,22 @@ void WebResourceLoadStatisticsStore::setLoginStatus(RegistrableDomain&& domain,
 {
     ASSERT(RunLoop::isMain());
 
-    if (loggedInStatus == IsLoggedIn::LoggedIn) {
-        auto loginStatusToSet = lastAuthentication && lastAuthentication->hasExpired() ? std::nullopt : std::optional(WTFMove(lastAuthentication));
-        if (loginStatusToSet)
-            loginStatusToSet->setTimeToLive(WebCore::LoginStatus::TimeToLiveLong);
-        auto pair = std::make_pair(loggedInStatus, loginStatusToSet);
-        m_loginStatus.set(domain, WTFMove(pair));
-    } else
-        m_loginStatus.remove(domain);
+    auto loginStatusToSet = lastAuthentication && lastAuthentication->hasExpired() ? std::nullopt : std::optional(WTFMove(lastAuthentication));
+    if (loginStatusToSet)
+        loginStatusToSet->setTimeToLive(WebCore::LoginStatus::TimeToLiveLong);
+
+    if (m_statisticsStore)
+        m_statisticsStore->setLoginStatus(WTFMove(domain), loggedInStatus, WTFMove(lastAuthentication), WTFMove(completionHandler));
     completionHandler();
 }
 
 void WebResourceLoadStatisticsStore::isLoggedIn(RegistrableDomain&& domain, CompletionHandler<void(bool)>&& completionHandler)
 {
     ASSERT(RunLoop::isMain());
 
-    auto it = m_loginStatus.find(domain);
-    completionHandler(it != m_loginStatus.end() && it->value.first == IsLoggedIn::LoggedIn);
```

Comment:
We should remove `m_loginStatus` since it is no longer being used.

Reply:
These lines have been removed actually

Reply:
I meant the variable that is still being stored on this object (`m_loginStatus` in the header).